canonicalwebteam/store_api/__init__.py

```diff
@@ -1 +1 @@
-__version__ = "1.0.1"
+__version__ = "1.0.2"
```

Comment:
We had a better way of doing this.... let me find it

Reply:
Done